completions/bash/oci-runtime-tool

```diff
@@ -119,6 +119,13 @@ __oci-runtime-tool_complete_log_level() {
 	" -- "$cur" ) )
 }
 
+__oci-runtime-tool_complete_compliance_level() {
+	COMPREPLY=( $( compgen -W "
+		must
+		should
```

Comment:
Lost 'may' in previous PR. We have ['MAY' in spec](https://github.com/opencontainers/runtime-spec/blob/master/config.md#platform-specific-configuration).  In this case, we 'may'  print 'MAY' information  when `platform.os is linux` but the `linux` item is not set.